css/mobile.css

```diff
@@ -182,10 +187,16 @@ button {
   background-position: center;
 }
 
+.skill-com > ul > li,
+.skill-com > h3,
+.input,
+.textarea {
+  padding: 10px;
+}
+
 .skill-com > ul > li {
   display: block;
   font: 400 22px/43px 'Inter', sans-serif;
-  padding: 10px;
 }
 
 .proj-com .proj-skill > ul > li {
```

Comment:
This seems too long. Let's just limit the selectors to two or a maximum of three classes or selectors.